package.json

```diff
@@ -154,7 +154,7 @@
         "@marko-tags/subscribe": "^0.5.1",
         "makeup-active-descendant": "0.6.1",
         "makeup-expander": "~0.10.1",
-        "makeup-floating-label": "~0.3.3",
+        "makeup-floating-label": "~0.4.1",
```

Comment:
Remove this, I will update dev deps on `14.0.0`

Reply:
Sounds good, undid the latest commit